### ISSUE TITLE
A couple suggestions for the optimization level

### ISSUE THREAD

**Issue by FabioPedretti** [amnuts/opcache-gui#67]
1. Optimizations should be shown with a bulleted list (or putting them inside a table), so it should be easier to read them (especially on smaller resolution screen, when the lines may break).
2. To have a better understanding of the optimizations, it would be nice to also show the disabled ones (eventually with different colors), in the order of the bitmask for configuring them.

I think these suggestions would also easie calculating the bitmask to make changes to the opcache configuration.

Thanks.

**Comment by amnuts** [comment]
Thanks for your input @FabioPedretti.  I've addressed some of this in the new release (3.2.1).

I've improved the optimisation output by putting then in a list with a little extra padding around.  I've hidden the bullet points, but they're legible enough as-is, I believe.

I can't really make the change to add the non-used optimisation levels  and colour them differently without a bit more of a major modification, and I'm not overly convinced myself it's needed.  But as a bit of a compromise I have added the bitmask value for the levels alongside their text output.  Would still need to calculate things, but might make it a little easier.

I'll consider the request for the non-used values for another time, though.
